### ISSUE TITLE
26-rc1 process_info garbage_collection missing max_heap_size key

### ISSUE THREAD

**Issue by gomoripeti** [erlang/otp#6922]
**Describe the bug**
On OTP 26-rc1 the key-value list returned by `erlang:process_info(Pid, garbage_collection).` only contains the value map, but not the key `max_heap_size`.

**To Reproduce**
On OTP 26-rc1
```
> erlang:process_info(self(), garbage_collection).
{garbage_collection,[#{error_logger => true,include_shared_binaries => false,
                       kill => true,size => 0},
                     {min_bin_vheap_size,46422},
                     {min_heap_size,233},
                     {fullsweep_after,65535},
                     {minor_gcs,4}]}
```

**Expected behavior**
On OTP 25
```
1> erlang:process_info(self(), garbage_collection).
{garbage_collection,[{max_heap_size,#{error_logger => true,kill => true,size => 0}},
                     {min_bin_vheap_size,46422},
                     {min_heap_size,233},
                     {fullsweep_after,65535},
                     {minor_gcs,4}]}
```

**Affected versions**
OTP 26-rc1

**Additional context**
Probably related to https://github.com/erlang/otp/pull/6345

(Sorry if this is already fixed, did not investigate latest changes on master)

**Comment by sverker** [comment]
Fixed in master for 26.0-rc2.
@gomoripeti Thanks for the report.

**Comment by gomoripeti** [comment]
thanks for the quick fix
